src/StoreApi/Routes/V1/OrderUpdateCustomer.php

```diff
@@ -0,0 +1,135 @@
+<?php
+namespace Automattic\WooCommerce\StoreApi\Routes\V1;
+
+use Automattic\WooCommerce\StoreApi\SchemaController;
+use Automattic\WooCommerce\StoreApi\Schemas\v1\AbstractSchema;
+use Automattic\WooCommerce\StoreApi\Utilities\OrderAuthorizationTrait;
+use Automattic\WooCommerce\StoreApi\Utilities\OrderController;
+use Automattic\WooCommerce\StoreApi\Utilities\UpdateCustomerTrait;
+
+/**
+ * OrderUpdateCustomer class.
+ *
+ * Updates the customer billing and shipping addresses and returns an updated order--taxes should be recalculated.
+ */
+class OrderUpdateCustomer extends AbstractRoute {
+	use OrderAuthorizationTrait;
+	use UpdateCustomerTrait {
+		get_args as protected get_update_customer_args;
+	}
+
+	/**
+	 * The route identifier.
+	 *
+	 * @var string
+	 */
+	const IDENTIFIER = 'order-update-customer';
+
+	/**
+	 * The schema item identifier.
+	 *
+	 * @var string
+	 */
+	const SCHEMA_TYPE = 'order';
+
+	/**
+	 * Order controller class instance.
+	 *
+	 * @var OrderController
+	 */
+	protected $order_controller;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param SchemaController $schema_controller Schema Controller instance.
+	 * @param AbstractSchema   $schema Schema class for this route.
+	 */
+	public function __construct( SchemaController $schema_controller, AbstractSchema $schema ) {
+		parent::__construct( $schema_controller, $schema );
+
+		$this->order_controller = new OrderController();
+	}
+
+	/**
+	 * Get the path of this REST route.
+	 *
+	 * @return string
+	 */
+	public function get_path() {
+		return '/order/(?P<id>[\d]+)/update-customer';
+	}
+
+	/**
+	 * Get method arguments for this REST route.
+	 *
+	 * @return array An array of endpoints.
+	 */
+	public function get_args() {
+		$args = $this->get_update_customer_args();
+
+		foreach ( $args as $index => $arg ) {
```

Comment:
Could you add a comment explaining this loop and why we're doing it? I get it but a comment would make it much quicker to understand when first coming to the code!

Reply:
Updated in [49d54f3](https://github.com/woocommerce/woocommerce-blocks/pull/9743/commits/49d54f337187530c649c360be3aec17a12cc5fb8).